extensions/widgets/clock/clock.lcb

```diff
@@ -85,6 +271,16 @@ private variable mCurrentTime 	as List
 private variable mRadius as Number
 private variable mCentre as List
 private variable mFontSize 		as Real
+private variable mDayNumberColor as Color
+private variable mNightNumberColor as Color
+private variable mDayHourHandColor as Color
+private variable mNightHourHandColor as Color
+private variable mDayMinuteHandColor as Color
+private variable mNightMinuteHandColor as Color
+private variable mDaySecondHandColor as Color
+private variable mNightSecondHandColor as Color
+private variable mDayFaceColor as Color
+private variable mNightFaceColor as Color
```

Comment:
It might be better to store all the color variables as `String`s instead of `Color`s, and that way you only need to convert to a color in one place (namely when painting) - it simplifies the getters, setters, and `OnLoad`, `OnSave` and `OnCreate` that way.